webapp/src/components/Metrics/PerformanceAnalysis.tsx

```diff
@@ -184,28 +191,40 @@ const PerformanceAnalysis: React.FC<Props> = ({ jobId, pipeline }) => {
       ...NUMBER_COL_DEF,
       field: "utteranceCount",
       headerName: "Utterance Count",
-      minWidth: 160,
+      minWidth: 150,
     },
     ...ALL_OUTCOMES.map<Column<Row>>((outcome) => ({
       ...NUMBER_COL_DEF,
       field: outcome,
       headerName: OUTCOME_PRETTY_NAMES[outcome],
+      minWidth: 105,
       renderHeader: () => OutcomeIcon({ outcome }),
-      valueGetter: ({ row }) => row.outcomeCount[outcome] / row.utteranceCount,
-      valueFormatter: percentageFormatter,
+      renderCell: ({ row }: GridCellParams<undefined, Row>) =>
+        VisualBarPercentage(
+          row.outcomeCount[outcome] / row.utteranceCount,
+          theme.palette[OUTCOME_COLOR[outcome]].main
+        ),
     })),
     ...customMetricNames.map<Column<Row>>((metricName) => ({
       ...NUMBER_COL_DEF,
       field: metricName,
       headerName: metricName,
-      valueGetter: ({ row }) => row.customMetrics[metricName],
-      valueFormatter: percentageFormatter,
+      minWidth: 105,
+      renderCell: ({ row }: GridCellParams<undefined, Row>) =>
+        VisualBarPercentage(row.customMetrics[metricName], "#d5d1e3"),
     })),
     {
       ...NUMBER_COL_DEF,
       field: "ece",
       headerName: "ECE",
-      valueFormatter: twoDigitFormatter,
+      minWidth: 105,
+      renderCell: ({ value }: GridCellParams) => (
+        <VisualBar
+          value={(value as number).toFixed(2)}
+          width={isNaN(value) ? 0 : value}
+          bgColor="#0b012e"
```

Comment:
This one is very close to
```suggestion
          bgColor={theme.palette.primary.dark}
```